configgen/src/main/java/com/yahoo/config/codegen/CppClassBuilder.java

```diff
@@ -357,7 +330,7 @@ void writeConfigClassFunctionDeclarations(Writer w, String className, CNode node
     }
 
     void writeStructFunctionDeclarations(Writer w, String className, CNode node, String indent) throws IOException {
-        w.write(indent + className + "(const std::vector<vespalib::string> & __lines);\n");
+        w.write(indent + className + "(const " + vectorTypeDefs.get("vespalib::string") + " & __lines);\n");
```

Comment:
Tangentially related to this PR; we should stop using identifiers containing double underscores. These are reserved for internal compiler/libc usage and I can't see any point in actually having them here. Just makes the code less readable, and since we seem to be using config input names verbatim anyway this doesn't help if a name collides with a reserved C++ keyword...

Reply:
Agree, but I think that belongs in a different PR